packages/forklift-console-plugin/src/modules/Plans/modals/PlanCutoverMigrationModal.tsx

```diff
@@ -71,7 +75,14 @@ export const PlanCutoverMigrationModal: React.FC<PlanCutoverMigrationModalProps>
     setCutoverDate(updatedFromDate.toISOString());
   };
 
-  const onTimeChange = (_event, _time, hour: number, minute: number) => {
+  const onTimeChange: (
+    event: React.FormEvent<HTMLInputElement>,
+    time: string,
+    hour?: number,
+    minute?: number,
+    seconds?: number,
+    isValid?: boolean,
+  ) => void = (event, time, hour, minute) => {
```

Comment:
```suggestion
  ) => void = (_event, _time, hour, minute) => {
```

Reply:
A good catch. Apparently the linter is not shouting on these cases for some reason.
Anyway, I fixed all found places, including ones that I didn't add as part of this PR.